tor_archivist/main.py

```diff
@@ -39,9 +56,12 @@ def run(config):
         if flair not in (css_flair.unclaimed, css_flair.completed):
             continue
 
+        # the original post that might have been transcribed
+        original_post = config.r.get_submission(post.url)
```

Comment:
Not sure about this, but doesn't PRAW need it to be `get_submission(url=...)` when you get a submission via URL?

Reply:
It does work without using the kw argument, but I agree it's better style. Will change.